app/main.py

```diff
@@ -1,4 +1,6 @@
-KNIGHTS = {
+from app.battle.battle import battle
+
+knights = {
```

Comment:
Place this variable in a separate file and name it `knight_config.py` to unload the `main.py` file. You could import this variable from the file as well. Besides that, it would be better to leave this variable in upper-case indicating that it is a constant.